sdk/math/storage/fee_tier.rs

```diff
@@ -0,0 +1,33 @@
+use crate::alloc::string::ToString;
+use crate::convert;
+use crate::errors::InvariantError;
+use crate::types::percentage::Percentage;
+use decimal::*;
+use serde::{Deserialize, Serialize};
+use tsify::Tsify;
+use wasm_bindgen::prelude::*;
+
+#[derive(Default, Debug, Copy, Clone, PartialEq, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub struct FeeTier {
+    pub fee: Percentage,
+    pub tick_spacing: u16,
+}
+
+#[wasm_bindgen(js_name = "newFeeTier")]
+pub fn new_fee_tier(js_fee: JsValue, js_tick_spacing: JsValue) -> Result<FeeTier, JsValue> {
```

Comment:
It will be better to use existing logic and the only thing handling in these functions are wrappers. So copy-past constructor from FeeTier and invoke `FeeTier::new()` here. 

It has not significant benefits now, but if we modify it so that the conversion is based on original code, it makes a significant difference.